### PR TITLE
refactor(templates/contribute/style.md): mention module docstrings

### PR DESCRIPTION
**Pull request by jcommelin**
Update the section on copyright headers and mention module docstrings.

### CODE REVIEW

File: templates/contribute/style.md [hunk 1]
```diff
@@ -39,12 +39,45 @@ break. You can also open namespaces in the same block.
...
+- `bar_of_foo`: a construction of a `bar`, given a `foo`.
+- `bar_eq`    : the main classification theorem of `bar`s.
+
+## Notation
+
+ - `|_|` : The barrification operator, see `bar_of_foo`.
+
+## References
+
+See [Thales600BC] for the original account on Xyzzyology.
```
Comment (gebner) [inline templates/contribute/style.md:79]: Do we have a `.bib` file?  Does it need to be updated?  Is it used by anything?

File: templates/contribute/style.md [hunk 2]
```diff
@@ -39,12 +39,45 @@ break. You can also open namespaces in the same block.
...
+```
```
Comment (jcommelin) [inline templates/contribute/style.md:80]:
```suggestion
```

New bibliography entries should be added to `docs/references.bib`.
```

File: templates/contribute/style.md [hunk 3]
```diff
@@ -39,12 +39,44 @@ break. You can also open namespaces in the same block.
...
+- `bar_of_foo`: a construction of a `bar`, given a `foo`.
+- `bar_eq`    : the main classification theorem of `bar`s.
+
+## Notation
+
+ - `|_|` : The barrification operator, see `bar_of_foo`.
+
+## References
+
+See [Thales600BC] for the original account on Xyzzyology.
```
Comment (jcommelin) [inline templates/contribute/style.md:79]:
````suggestion
See [Thales600BC] for the original account on Xyzzyology.
```

New bibliography entries should be added to `docs/references.bib`.
````

File: templates/contribute/style.md (hunk 1 above)
Comment (jcommelin) [reply templates/contribute/style.md:79]: Added a pointer to the `.bib` file

**Comment by bryangingechen** [comment]
Would it make sense to link to the doc requirements page instead / as well: https://leanprover-community.github.io/contribute/doc.html

**Comment by jcommelin** [comment]
Sure... feel free to push a sentence.

File: templates/contribute/style.md [hunk 4]
```diff
@@ -39,12 +39,47 @@ break. You can also open namespaces in the same block.
...
+```
+
+New bibliography entries should be added to `docs/references.bib`.
```
Comment (bryangingechen) [inline templates/contribute/style.md:83]:
```suggestion

See our [documentation requirements](doc.html) for more suggestions and examples.

```

**Pull request merged**